### PR TITLE
sched: Better to track numRunning than numAvail nodes. Cleaned up cluster logging.

### PR DESCRIPTION
**Pull request by jasonbs10**

### CODE REVIEW

**Comment by JeanetteBruno** [comment]
while we are in here.... can we change clusterState.numAvail to numIdle.   We report the number of nodes (len(c.nodes)) against the the 'availableNodes' label and c.numAvail against the 'idleNodes' label - this is confusing.

**Comment by jasonbs10** [comment]
I'll rename but how about numFree?

File: common/stats/stats_names.go [hunk 1]
```diff
@@ -84,9 +84,14 @@ const (
...
 
 	/*
-		the number of idle worker nodes (available, but not running a task)
+		the number of free worker nodes (available, but not running a task)
 	*/
-	ClusterIdleNodes = "idleNodes"
+	ClusterFreeNodes = "freeNodes"
+
+	/*
+		the number of running worker nodes (running + avail + suspended == allNodes)
```
Comment (JeanetteBruno) [inline common/stats/stats_names.go]: running + free + suspended

File: sched/scheduler/cluster_state.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ type clusterState struct {
 	maxLostDuration  time.Duration                 // after which we remove a node from the cluster entirely
 	maxFlakyDuration time.Duration                 // after which we mark it not flaky and put it back in rotation.
 	readyFn          ReadyFn                       // If provided, new nodes will be suspended until this returns true.
-	numAvail         int                           // Number of free nodes that are not in a suspended state.
+	numRunning       int                           // Number of running nodes. running + avail + suspended == allNodes
```
Comment (JeanetteBruno) [inline sched/scheduler/cluster_state.go]: running +  free + suspended

**Pull request merged**